controllers/usersignup/usersignup_controller_test.go

```diff
@@ -2490,6 +2473,11 @@ func TestUserSignupReactivateAfterDeactivated(t *testing.T) {
 		Spec: toolchainv1alpha1.UserSignupSpec{
 			Userid:   "UserID123",
 			Username: meta.Name,
+			IdentityClaims: toolchainv1alpha1.IdentityClaimsEmbedded{
+				PropagatedClaims: toolchainv1alpha1.PropagatedClaims{
+					Email: "john.doe@redhat.com",
+				},
+			},
```

Comment:
I know that without the email address being set here, the black box mystery that is the metrics tests will fail.  I've spent some time trying to work out why this is the case, and my only suspicion is that it's related to the email domain function.

Reply:
Talking about notifications - I see that the e2e tests are failing for the idling notifications. My guess is that it's because the idler controller is still using the email annotation for sending the notification. You need to update that part first before dropping it from the MUR: 
https://github.com/codeready-toolchain/member-operator/blob/f4220407cbae99fa7d3713504fef5f7dbd196ace/controllers/idler/idler_controller.go#L270-L271

Reply:
@MatousJobanek thanks for pointing this out, I wasn't aware of the idler controller or what it did.  I've updated it to now use the propagated claims.